packages/components/src/UserCard/UserCard.d.ts

```diff
@@ -14,6 +14,8 @@ export declare type UserCardProps = {
         flag?: string;
         phone?: string;
         profileUrl?: string;
+        targetBlank?: boolean;
```

Comment:
vllt lieber etwas wie 
```suggestion
        link: {
          target?: string;
          title: string;
        }
```

Reply:
macht Sinn, stimmt!